composer.json

```
@@ -40,6 +40,7 @@
         "phpstan/phpstan": "1.12.7",
         "phpunit/phpunit": "10.5.36",
         "phing/phing": "3.0",
+        "rector/rector": "^1.2.7",
```

We should remove the caret here:

```suggestion
        "rector/rector": "1.2.7",
```